src/main/java/com/starkbank/BoletoHolmes.java

```diff
@@ -0,0 +1,384 @@
+
+    public BoletoHolmes(Map<String, Object> data) throws Exception {
+        super(null);
+        HashMap<String, Object> dataCopy = new HashMap<>(data);
+
+        this.tags = (String[]) dataCopy.remove("tags");
+        this.boletoId = (String) dataCopy.remove("boletoId");
+        this.status = (String) dataCopy.remove("status");
+        this.result = (String) dataCopy.remove("result");
+        this.created = (String) dataCopy.remove("created");
+        this.updated = (String) dataCopy.remove("updated");
+
+        if (!dataCopy.isEmpty()){
+            throw new Exception("Unknown parameters used in constructor: [" + String.join(", ", dataCopy.keySet()) + "]");
+        }
+    }
+
+    /**
+     * Retrieve a specific BoletoHolmes
+     * <p>
+     * Receive a single BoletoHolmes object previously created by the Stark Bank API by passing its id
+     * <p>
+     * Parameters (required):
+     * @param id [string]: object unique id. ex: "5656565656565656"
+     * Return:
+     * @return BoletoHolmes object with updated attributes
+     * @throws Exception error in the request 
+     */
+    public static BoletoHolmes get(String id) throws Exception {
+        return BoletoHolmes.get(id, null);
+    }
+
+    /**
+     * Retrieve a specific BoletoHolmes
+     * <p>
+     * Receive a single BoletoHolmes object previously created by the Stark Bank API by passing its id
+     * <p>
+     * Parameters (required):
+     * @param id [string]: object unique id. ex: "5656565656565656"
+     * <p>
+     * Parameters (optional):
+     * @param user [Project object]: Project object. Not necessary if starkbank.user was set before function call
+     * Return:
+     * @return BoletoHolmes object with updated attributes
+     * @throws Exception error in the request 
+     */
+    public static BoletoHolmes get(String id, Project user) throws Exception {
+        return Rest.getId(data, id, user);
+    }
+
+    /**
+     * Create BoletoHolmes
+     * <p>
+     * Send a list of BoletoHolmes objects for creation in the Stark Bank API
+     * <p>
+     * Parameters (required):
+     * @param holmes [list of BoletoHolmes objects]: list of BoletoHolmes objects to be created in the API
+     * <p>
+     * Parameters (optional):
+     * @param user [Project object]: Project object. Not necessary if starkbank.user was set before function call
+     * <p>
+     * Return:
+     * @return list of BoletoHolmes objects with updated attributes
+     * @throws Exception error in the request
+     */
+    @SuppressWarnings("unchecked")
+    public static List<BoletoHolmes> create(List<?> holmes, Project user) throws Exception {
+        List<BoletoHolmes> holmesList = new ArrayList<>();
+        for (Object sherlock : holmes){
+            if (sherlock.getClass() == HashMap.class){
+                holmesList.add(new BoletoHolmes((Map<String, Object>) sherlock));
+                continue;
+            }
+            if (sherlock.getClass() == BoletoHolmes.class){
```

Comment:
aqui a opção certa é if(sherlock instaceof BoletoHolmes)

Reply:
Roger that.

Reply:
Também tá certo do jeito que tá, mas entendi seu ponto no comentário debaixo.